www/src/examples/CloseButton/Variants.js

```diff
@@ -0,0 +1,4 @@
+<div className="bg-dark">
```

Comment:
```suggestion
<div className="bg-dark p-3">
```
I think some extra padding would look nicer so the buttons aren't at the edges

Reply:
Sounds good -- I debated doing this but didn't want to add too much to the example other than the CloseButton itself